packages/itwinui-css/backstop/tests/input.html

```diff
@@ -502,5 +505,118 @@ <h2>No wrapping label</h2>
       </div>
 
     </section>
+
+    <br />
+
+    <h2>Search box</h2>
+    <section id="demo-search-box">
+
+      <h3>Current method</h3>
+      <div class="iui-input-container iui-inline-icon">
+        <input
+          placeholder="Placeholder…"
+          type="search"
+          class="iui-input"
+          id="test-1"
+        />
+        <svg-search
+          class="iui-input-icon"
```

Comment:
how will this `iui-input-icon` class be applied in react?

Reply:
This class in not needed.